Tutorials/Creating-a-Custom-Dashboard/index.md

```diff
@@ -198,7 +224,7 @@ If all is setup fine we should now receive the 'Hello world' alert every time th
 
 ### Going further - Umbraco Angular Services and Directives
 
-Umbraco has a fine selection of angular directives, resources and services that you can use in your custom property editors and dashboards, the details are here: https://our.umbraco.com/apidocs/v7/ui/#/api
+Umbraco has a fine selection of angular directives, resources and services that you can use in your custom property editors and dashboards, the details are here: https://our.umbraco.com/apidocs/v8/ui/#/api
 
```

Comment:
Line 229:
"Umbraco is a place where everybody knows your name..." - a bit creepy if you ask me.

Reply:
Haha that's fair, it's from the original v7 tutorial, I didn't change it, but will remove that part for now 😁

Reply:
https://www.youtube.com/watch?v=h-mi0r0LpXo